modules/packages/npm/metadata.go

```diff
@@ -23,5 +23,5 @@ type Metadata struct {
 	OptionalDependencies    map[string]string `json:"optional_dependencies,omitempty"`
 	Bin                     map[string]string `json:"bin,omitempty"`
 	Readme                  string            `json:"readme,omitempty"`
-	Repository              Repository        `json:"repository,omitempty"`
+	Repository              Repository        `json:"repository"`
```

Comment:
Also, not sure on these.

Reply:
gopls is right again.

"struct" can't be empty, "omitempty" does nothing.

Reply:
Thanks for this insight :)